pydatastructs/linear_data_structures/arrays.py

```diff
@@ -283,10 +368,11 @@ class ArrayForTrees(DynamicOneDimensionalArray):
 
     pydatastructs.linear_data_structures.arrays.DynamicOneDimensionalArray
     """
+
     def _modify(self):
-        if self._num/self._size < self._load_factor:
+        if self._num / self._size < self._load_factor:
```

Comment:
`_modify` can be removed. The concept of load_factor isn't that easy to apply here.

Reply:
it is not from the multi dimensional array, should I still change it??

Reply:
```suggestion
        if self._num/self._size < self._load_factor:
```